code/__DEFINES/research.dm

```diff
@@ -61,3 +61,21 @@
 #define DESIGN_ID_IGNORE "IGNORE_THIS_DESIGN"
 
 #define RESEARCH_MATERIAL_RECLAMATION_ID "__materials"
+
+//When adding new types, update the list below!
+#define TECHWEB_POINT_TYPE_GENERIC "General Research"
+
+#define TECHWEB_POINT_TYPE_DEFAULT TECHWEB_POINT_TYPE_GENERIC
+
+//Formatted this way for readability.
+#define TECHWEB_POINT_TYPE_LIST list(\

```

Comment:
What are these three defines for? Only one of them is even used, can it just be inlined onto `SSresearch` where `point_types` is declared?